cmd/common.go

```diff
@@ -111,3 +134,28 @@ func MakeDirs(devPath string) error {
 	}
 	return nil
 }
+
+func Die(message string) { 
+  fmt.Printf(message)
+  os.Exit(1)
+}
+
+func GolangHolochainDir(subPath ...string) string {
+  joinable := append([]string{os.Getenv("GOPATH"), "src/github.com/metacurrency/holochain", }, subPath...)
```

Comment:
I  think that filepath rewrites the string, since they have those kinds of strings in their examples

Im just making it up from the examples though.